utils/use_require.js

```diff
@@ -70,29 +69,26 @@ const babelTransformFile = promisify(babel.transformFile);
 
 // walkDir *recursively* walks directories trees,
 // calling the callback for all normal files found.
-const walkDir = function (base_path, cb, filter) {
+function walkDir(base_path, cb, filter) {
     return readdir(base_path)
-    .then(files => {
+    .then((files) => {
```

Comment:
Not needed, right?

Reply:
It's the convention that parenthesis are skip on 1-lines but used when using braces.

`myArg => myFunc(myArgs)`

vs

```
(myArg) => {
  return myFunc(myArgs);
}
```

Would you prefer something else?